src/util/Enums.js

```diff
@@ -40,6 +40,12 @@ define({
   // Operations
   PRIMARY_AUTH: 'PRIMARY_AUTH',
   FORGOT_PASSWORD: 'FORGOT_PASSWORD',
-  UNLOCK_ACCOUNT: 'UNLOCK_ACCOUNT'
+  UNLOCK_ACCOUNT: 'UNLOCK_ACCOUNT',
+
+  // Auth Types
+  SESSION_STEP_UP: 'SESSION_STEP_UP',
```

Comment:
API will not return SESSION_SSO just yet, but it is in the roadmap. So for now I will just add it to our response object.